src/modules/px4iofirmware/protocol.h

```diff
@@ -242,6 +242,8 @@ enum {							/* DSM bind states */
 
 #define PX4IO_P_SETUP_SBUS_RATE			22	/* frame rate of SBUS1 output in Hz */
 
+#define PX4IO_P_SETUP_MOTOR_SLEW_MAX 	24 	/* max motor slew rate */
```

Comment:
@julianoes That is reserved for the thrust curve PR, which is also up. Wanted to avoid merge conflicts.